Just use message as @message

railsbros-dirk opened the pull request:
The `@message` field contains information already present in the JSON itself. I think that information should not be duplicated and especially not if the information is already present in structured way.

Additionally I

* refactored the message creation code a bit
* added a spec for the status `dead`
* made spacing in the specs consistent
* added a Rake task for running specs and made it the default

timflapper commented:
@railsbros-dirk, unfortunately this is not compatible with Ruby versions <  2.2 because of the hash notation using `:` with string keys. For the rest it seems like a sensible update. If you change the hash notation I think we can merge this.

railsbros-dirk commented:
Hi Tim, thanks for the feedback. Sorry for the hash syntax. Didn't thought about it. I changed it and rebased on master. You should have another look.

> On 06.04.2016, at 11:58, Tim Flapper <notifications@github.com> wrote:
> 
> @railsbros-dirk, unfortunately this is not compatible with Ruby versions < 2.2 because of the hash notation using : with string keys. For the rest it seems like a sensible update. If you change the hash notation I think we can merge this.
> 
> ―
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>

timflapper commented:
Did a quick test locally and looks good to me so far. Merging.
